src/app/utils/utils.ts

```diff
@@ -17,9 +17,9 @@ export class Utils {
 
   public static wrapCss(cssDeclaration: IDeclaration, useInspect: boolean): string {
     if (useInspect) {
-      return `${WRAPPER_CSS_ID}.${cssDeclaration.name}{content:"#{inspect(${cssDeclaration.value})}";}`;
+      return `${WRAPPER_CSS_ID}._${cssDeclaration.name}{content:"#{inspect(${cssDeclaration.value})}";}`;
```

Comment:
Underscore is required to avoid following error:

```
Error: Expected identifier.
   ╷
46 │ #sass-export-id.100{content:"#{inspect(80%)}";}
   │                 ^
   ╵

```